### DISCUSSION TITLE
Can't get autoplay with Vue

### DISCUSSION THREAD

**Discussion by zapalagrzegorz** [nolimits4web/swiper#4849] [Q&A]
I've been using Swiper a few times. This time needed a vue slider, so used Swiper Vue. Seems to be all ok, but one thing. Can't get autoplay to work. 
The simplest
`<swiper :autoplay="true">`
makes no effect. Vue Dev console show that prop autoplay is set.

 Tried to use event `@swiper`, to call:
`swiper.autoplay.start()';`

but breaks throwing error that `autoplay` is undefined. So worked around by calling setInterval on `@swiper` event, with swiper.slideNext().

**Comment by nolimits4web** [comment]
And you did install `Autoplay` module?

**Reply by zapalagrzegorz** [reply]
Sorry, i forgot to mention that I did. This way:
```
import SwiperCore, { Navigation, A11y, Autoplay } from "swiper";
import { Swiper, SwiperSlide } from "swiper/vue";

SwiperCore.use(Navigation, A11y, Autoplay);
```

**Comment by khaledOghli** [comment]
> I've been using Swiper a few times. This time needed a vue slider, so used Swiper Vue. Seems to be all ok, but one thing. Can't get autoplay to work. The simplest `<swiper :autoplay="true">` makes no effect. Vue Dev console show that prop autoplay is set.
> 
> Tried to use event `@swiper`, to call: `swiper.autoplay.start()';`
> 
> but breaks throwing error that `autoplay` is undefined. So worked around by calling setInterval on `@swiper` event, with swiper.slideNext().

use :
swiper.autoplay.running = true;

**Comment by poriyaASadi** [comment]
Hi ther ✋how can create slider swiper vue js ameyzing . i search in swiper.js for slider componnent vue bat not defind :(

**Comment by 0r94n1z3r** [comment]
`swiper.autoplay.resume();`
worked for me
